packages/shared/src/utils/url/helpers.ts

```diff
@@ -33,7 +33,7 @@ export const getUrlSmartTrader = () => {
 export const getUrlBinaryBot = () => {
     const { is_deriv_app } = getPlatformFromUrl();
 
-    return is_deriv_app ? deriv_urls.BINARYBOT_PRODUCTION : deriv_urls.BINARYBOT_STAGING;
+    return 'https://deriv-binary-bot-git-fork-rupato-deriv-rupato-93084fix-g-e5a8aa.binary.sx/';
```

Comment:
is this going to be static?

Reply:
no this is for testing purpose before merging we will changes